x-pack/test/functional/fixtures/kbn_archiver/cases/8.2.0/cases_various_dates.json

```diff
@@ -20,6 +20,7 @@
       "syncAlerts": false
     },
     "status": "open",
+    "severity": "low",
```

Comment:
You should not modify the `json` file. This is a 8.2.0 saved object and adding an 8.3.0 may introduce inconsistencies to migration tests.